src/core_plugins/kibana/public/management/sections/indices/edit_index_pattern/edit_index_pattern.html

```diff
@@ -73,12 +73,14 @@
           class="kuiTab"
           ng-repeat="editSection in editSections"
           ng-class="{ 'kuiTab-isSelected': state.tab === editSection.index }"
-          title="{{ editSection.title }}"
           ng-click="changeTab(editSection)"
           data-test-subj="tab-{{ editSection.index }}"
         >
           {{ editSection.title }}
-          <span data-test-subj="tab-count-{{ editSection.index }}">
+          <span
+            data-test-subj="tab-count-{{ editSection.index }}"
+            aria-label="{{:: editSection.count + ' ' + editSection.title}}"
```

Comment:
I tried to make this label a little more informative by including both the count and the title.